src/entities/mellow-stateless/getters/optimisers/getOptimisersInfo.ts

```diff
@@ -0,0 +1,45 @@
+import { ethers } from 'ethers';
```

Comment:
I's change the name of this function& file `getAllOptimisersInfo`, such that we don't get it confused with `getOptimiserInfo`

Reply:
https://github.com/Voltz-Protocol/v1-sdk/pull/188/commits/c589f05e4a6a54fd84e3b201d352a2d1316d73f1